tests/test_set_grpc_max_size_in_bytes.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Ant Group Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import multiprocessing
+import pytest
+import fed
+
+@fed.remote
+class My:
+    def __init__(self) -> None:
+        self._val = ""
+
+    def set_val(self, val):
+        self._val += val
+        return self._val
+
+
+@fed.remote
+def concat_str(val1, val2):
+    return f"{val1} {val2}"
+
+def run(party):
+    cluster = {
+        'alice': {'address': '127.0.0.1:11010'},
+        'bob': {'address': '127.0.0.1:11011'},
+    }
+    fed.init(address='local', cluster=cluster, party=party, cross_silo_messages_max_size_in_bytes=10)
+
+    # The given strings are too long for cross_silo_messages_max_size_in_bytes
+    actor_alice = My.party("alice").remote()
+    actor_bob = My.party("bob").remote()
+    val_alice = actor_alice.set_val.remote("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.")
+    val_bob = actor_bob.set_val.remote("Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")
+
+    concatenated_val_obj = concat_str.party("bob").remote(val_alice, val_bob)
```

Comment:
[self-review]
I was expecting gRPC communication to occur on this line, but apparently it does not.

Reply:
No, It is working ... but my ex-PR(#32) is broken.

``` console
(SendProxyActor pid=22207) INFO:fed.barriers:[('grpc.max_send_message_length', 524288000), ('grpc.max_receive_message_length', 524288000), ('grpc.enable_retries', 1), ('grpc.service_config', '{"methodConfig": [{"name": [{"service": "GrpcService"}], "retryPolicy": {"maxAttempts": 5, "initialBackoff": "5s", "maxBackoff": "30s", "backoffMultiplier": 2, "retryableStatusCodes": ["UNAVAILABLE"]}}]}')]
```